Sources/TokamakLint/TokamakLint.swift

```diff
@@ -0,0 +1,115 @@
+                      print("directoryEnumerator error at \(url): ", error)
+                      return true
+        })!
+      for case let fileURL as URL in enumerator {
+        if isSwiftFile(fileURL.path) {
+          let fileSource = try String(contentsOf: fileURL, encoding: .utf8)
+          if hasTokamakImport(fileSource) {
+            print(fileURL.path)
+            lintPropsEquatable(fileURL.path)
+          }
+        }
+      }
+    } catch {
+      print(error)
+    }
+  }
+
+  public func lintFile(_ path: String) {
+    do {
+      let fileURL: URL = URL(fileURLWithPath: path)
+      let parsedTree = try SyntaxTreeParser.parse(fileURL)
+      let visitor = TokenVisitor()
+      parsedTree.walk(visitor)
+      let structs = visitor.getNodes(get: "StructDecl", from: visitor.tree[0])
+      for structNode in structs {
+        let isInherited = visitor.isInherited(
+          node: structNode,
+          from: "Equatable"
+        )
+        print(isInherited)
+      }
+    } catch {
+      print(error)
+    }
+  }
+
+  public func isPropsEquatable(_ path: String) -> Bool {
+    var res: Bool = false
+    do {
+      let fileURL: URL = URL(fileURLWithPath: path)
+      let parsedTree = try SyntaxTreeParser.parse(fileURL)
+      let visitor = TokenVisitor()
+      parsedTree.walk(visitor)
+      let structs = visitor.getNodes(get: "StructDecl", from: visitor.tree[0])
+      for structNode in structs where structNode.children[1].text == "Props" {
+        res = visitor.isInherited(node: structNode, from: "Equatable")
+        if !res {
+          print("\(path):",
+                "\(structNode.range.startRow):",
+                "\(structNode.range.startColumn):",
+                " warning: Props is not Equatable")
+        }
+      }
+    } catch {
+      print(error)
+      return false
+    }
+    return res
+  }
+
+  public func lintPropsEquatable(_ path: String) {
+    do {
+      let fileURL: URL = URL(fileURLWithPath: path)
+      let parsedTree = try SyntaxTreeParser.parse(fileURL)
+      let visitor = TokenVisitor()
+      parsedTree.walk(visitor)
+      let structs = visitor.getNodes(get: "StructDecl", from: visitor.tree[0])
+      for structNode in structs where structNode.children[1].text == "Props" {
+        if !visitor.isInherited(node: structNode, from: "Equatable") {
+          print("\(path):",
+                "\(structNode.children[1].range.startRow + 1):",
+                "\(structNode.children[1].range.startColumn + 1):",
+                " warning: Props is not Equatable:",
+                " add conformance to Equatable protocol to your Props type")
+        } else {
+          print(path)
+        }
+      }
+    } catch {
+      print(error)
+    }
+  }
+
+  public func isSwiftFile(_ path: String) -> Bool {
+    return path.contains(".swift")
+  }
+
+  public func hasTokamakImport(_ code: String) -> Bool {
+    return code.contains("import Tokamak")
```

Comment:
is there a more reliable way to check for `import` syntax nodes? This function would return `true` even for code that has `import Tokamak` commented out